alacritty/src/display.rs

```diff
@@ -142,6 +142,13 @@ impl Display {
         // Spawn window
         let mut window = Window::new(event_loop, &config, size)?;
 
+        #[cfg(not(any(target_os = "macos", windows)))]
+        {
+            if event_loop.is_wayland() {
+                window.set_wayland_theme(&config);
+            }
+        }
```

Comment:
Is there a reason these aren't marked with a `#[cfg(feature = "wayland")]`?

Reply:
well, there's no reason to opt out specific features code, since it doesn't affect compilation.